script/build.ts

```diff
@@ -56,6 +36,10 @@ function makeDistributedCLIExecutable() {
   shell.chmod("+x", cli);
 }
 
+function bundleDistributables() {
+  shell.cp(["readme.md", "package.json"], "dist");
+}
```

Comment:
`README.md` is not all lower-case.  This will not work on Linux.

Reply:
Hmm, I changed the filename but git didn't notice. Will fix.